Proxy/proxy.sample.ini

```diff
@@ -6,9 +6,16 @@ PORT=8000
 FILENAME=log.db
 SCHEMANAME=db.sql
 
+[TESTDATABASE]
+FILENAME=log.test.db
+
 [PROXY]
 LOG=0
 UNITS=1
+TESTMODE=0
+TESTRATE=1
+TESTCALLSIGN=TESTCS
```

Comment:
Change sample configuration file callsign & nodeid to `REPLACEME` since it's standard to need to replace them if you are to use it.